src/tools/store.cpp

```diff
@@ -27,7 +27,11 @@ void deleteRegion(const storage::SharedRegionRegister::ShmKey key)
 
 void listRegions()
 {
-
+    if (!storage::SharedMonitor<storage::SharedRegionRegister>::exists())
+    {
+        osrm::util::Log() << "No shared memory regions found. Try running osrm-datastore";
```

Comment:
What about just printing an empty list in this case? That would make the output more predictable.